src/plugins/genome_aligner/src/GenomeAlignerTask.cpp

```diff
@@ -45,6 +45,7 @@
 
 namespace U2 {
 
+//TODO: calling tr() int static context! Translator is not initialized yet!
```

Comment:
Plugins are loaded after the translator initialized: it may be safe for them use this kind of code.
But in general (for core-libs) this style is very unsafe.